R/tidy_add_reference_rows.R

```diff
@@ -186,11 +186,11 @@ tidy_add_reference_rows <- function(x, model = tidy_get_model(x)) {
   }
 
   if (!has_var_label)
-    x <- x %>% dplyr::select(-.data$var_label)
+    x[["var_label"]] <- NULL
```

Comment:
FYI, I made this update because the model attribute was being stripped from the tidy tibble.

Reply:
Thanks a lot for your proposal and more in general for testing the different features. Do you mind to be added as a contributor?

Regarding mice objects, I'm not sure that tidy_and_attach is the good entry point as the model could be unattached and passed explicitly to all functions.

Would it be better to attach the full mice object and then to add specific methods for the model_* functions?

Reply:
That makes sense.  There is already a method fn for model.frame.  I'll need to add one for `stats::terms()` and the fn that gets the contrasts as well.

Happy to be a contributor here.  It's an awesome project!